slack-user.c

```diff
@@ -148,13 +148,27 @@ void slack_user_retrieve(SlackAccount *sa, const char *uid, SlackUserCallback *c
 	slack_api_call(sa, user_retrieve_cb, lookup, "users.info", "user", uid, NULL);
 }
 
+static void self_presence_cb(SlackAccount *sa, gpointer data, json_value *json, const char *error) {
+	if (json_get_prop_boolean(json, "manual_away", TRUE))
+		return;
+
+	if (sa->ping_timer) {
+		purple_timeout_remove(sa->ping_timer);
```

Comment:
Should this 'clear if exists' code be in the `rtm_connect_cb()` func, along with the `sa->rtm` clearout?

Reply:
Based on `slack-rtm.c:187`, yes. Addressed in #100